### ISSUE TITLE
Data model and ownership

### ISSUE THREAD

**Issue by mr-sudaca** [mr-sudaca/yama#19]
The idea behind this PR is to addres data modeling (mayo to many relationships) and also to address ownership validation over albums, artists, songs and playlists

https://github.com/mr-sudaca/yama/pull/18

**Comment by alex-nexus** [comment]
can you explain the purpose of owner for album?

**Comment by mr-sudaca** [comment]
@alex-nexus yeah, I just wanted to follow the same ownership pattern across all models. Maybe albums can be modified by anyone, but that could be a feature discussion
